uri-scheme.md

```diff
@@ -0,0 +1,35 @@
+# URI Scheme
+
+The `wormhole` URI Scheme is used to encode a wormhole code as a URI. This can then be used to generate QR codes, or be opened by the platform URI handler to open a supporting client.
+
+The general format looks like this, and assumes default values for all query fields:
+`wormhole:{code}`
+
+It can be extended by appending a [query string](https://en.wikipedia.org/wiki/Query_string).
+
+## Query fields
+Applications MUST parse all query fields specified below and fail if they contain unknown fields or unsupported parameter values. Query values are [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding).
+
+* `version`: The version of the URI scheme.  No other value than `0` is accepted.
+Default: 0
+
+* `rendezvous`: The rendezvous server and protocol to use, including its port.
+Default: `ws://relay.magic-wormhole.io:4000/v1`
```

Comment:
The `/v1` must not be part of the URL, and instead must be added by the client implementation. The assumption is that a rendezvous server may have different versioned endpoints that are all compatible with each other.